build.sh

```diff
@@ -461,8 +461,8 @@ if $__buildnative; then
     prepare_native_build
 
     # Build the corefx native components.
-
-    build_native_corefx
+    # WCF does not have a native component
+    # build_native_corefx
```

Comment:
LGTM

Nit: I believe @StephenBonikowsky is using this comment convention to make these easier to spot:
```
  # *** start WCF Content ***
```